ansible/avena/roles/can/tasks/can.yml

```diff
@@ -0,0 +1,48 @@
+- name: ensure CAN interfaces
+  when: oats_can_interfaces is defined
+  block:
+    - name: ensure can interface bitrate and options are set
+      become: yes
+      template:
+        dest: /etc/can.conf
+        src: config/can.j2
+        owner: root
+        group: root
+        mode: "0644"
+
+    - name: ensure can interfaces are enabled and configured
+      become: yes
+      copy:
+        dest: /etc/udev/rules.d/
+        src: udev/60-can.rules
+        owner: root
+        group: root
+        mode: "0644"
+      notify: reload udev
+
+    - name: ensure can@ service is installed
+      become: yes
+      copy:
+        dest: /etc/systemd/system/can@.service
+        src: systemd/can@.service
+        owner: root
+        group: root
+        mode: "0644"
+      notify: reload systemd
+
+    - name: ensure can interfaces are started
+      become: yes
+      systemd:
+        name: can@{{ item }}
+        state: started
+      with_items: "{{ oats_can_interfaces }}"
```

Comment:
You are still using the old `oats_can_interfaces` variable  instead of `can_interfaces` here and on line two which keeps the configuration from working properly

Reply:
Thanks for pointing that out. I just fixed this and it worked in one ISOBlue I just deployed. More tests will have to be in order to confirm this works